packages/appstore/src/components/wallet-header/wallet-header-title.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TWalletSvgCurrency } from 'Types';
+
+type TWalletHeaderTitle = {
+    is_demo: boolean;
+    currency: TWalletSvgCurrency;
+    jurisdiction: 'svg' | 'malta';
```

Comment:
How about bvi and labuan ?

Reply:
![Uploading image.png…]()

Reply:
@thisyahlen-deriv Figma template has only SVG and MALTA wallets. Also in wallets descriptions I didn't see any other jurisdictions. At this moment I'm sure we need just these 2 cases

Reply:
I think the design doesnt cover all types of jurisdiction. But its better for you to confirm in the channel as well 🙏

Reply:
@thisyahlen-deriv I have asked Nader and received a confirmation we have only 2 wallets jurisdictions right now (svg and malta)

Reply:
@sergei-deriv but in future we will add the other jurisdictions, let's add all of them to remove further efforts later on

Reply:
@sergei-deriv I believe what  we have here is `landing_compony_shortcode` and not jurisdiction.  `'matltainvest' and 'iom' ` were options for shortcode but it's not there any more, so we can leave it like it is.
BTW please change the name to `shortcode` instead of `jurisdiction`